FilePersistence/src/version_control/merge/ListMergeComponentV2.cpp

```diff
@@ -74,7 +74,23 @@ ListMergeComponentV2::IdToIndexMap ListMergeComponentV2::computeAdjustedIndices(
 {
 	IdToIndexMap map;
 	auto chunks = listToChunks(listId, mergeData);
-	(void) chunks;
+	int index = 0;	// Final index of elements in the merged tree such that all labels are unique
+	for (auto chunk : chunks)
+	{
+		if (chunk->stable_)
+			for (auto id : chunk->spanBase_)
+			{
+				map.insert(id, qMakePair(QString::number(index), 0));
+				index++;
+			}
+		else
+		{
+			IdPositionMap temp;	// temp will store the relative positions of elements of unstable chunk
```

Comment:
don't call things `temp` That names is useless since it doesn't give you any information. Provide a better name.